src/test/util/mod.rs

```diff
@@ -87,32 +87,23 @@ impl TestClient {
         ))
         .unwrap();
 
-        let mut server_version = None;
-        let mut server_parameters = None;
+        let build_info = RunCommand::new("test".into(), doc! { "buildInfo":  1 }, None).unwrap();
 
-        if collect_server_info {
-            let build_info =
-                RunCommand::new("test".into(), doc! { "buildInfo":  1 }, None).unwrap();
-
-            let response = client
-                .execute_operation(build_info, &mut session)
-                .await
-                .unwrap();
-
-            let info: BuildInfo = bson::from_bson(Bson::Document(response)).unwrap();
-            let server_version_str = info.version.split('-').next().unwrap();
-            server_version = Some(Version::parse(server_version_str).unwrap());
+        let response = client
+            .execute_operation(build_info, &mut session)
+            .await
+            .unwrap();
 
-            let get_parameters =
-                RunCommand::new("admin".into(), doc! { "getParameter": "*" }, None).unwrap();
+        let info: BuildInfo = bson::from_bson(Bson::Document(response)).unwrap();
+        let server_version_str = info.version.split('-').next().unwrap();
+        let server_version = Version::parse(server_version_str).unwrap();
 
-            // The command above may fail due to insufficient permissions. In that case, the unified
-            // test runner will skip any tests with a serverParameters runOnRequirement as the check
-            // will fail.
```

Comment:
It wasn't clear when this actually failed and we were unwrapping it anyways in the run on requirement, so I just removed this handling and unwrapped here.